hack/logcollector/main.go

```diff
@@ -0,0 +1,13 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package main
+
+import "github.com/edgelesssys/constellation/v2/hack/logcollector/cmd"
+
+func main() {
+	cmd.Execute()
+}
```

Comment:
Even though it's a hack, can you write a very short README.md with purpose and example usage?

I could imagine it being useful even during local development and it would be nice to have a quick reference how to deploy the charts (kind of like in the GH action)

Reply:
I added docs [here](https://github.com/edgelesssys/constellation/pull/2080/files#diff-a2e3245686df61d5cc57d3bdabb74eb4c4fde1000f868577d6aef52ccf08469b)

Reply:
Your os.Exit is in the cmd package, which is an antipattern. Either move it up into main package, or move cmd -> cmd/logcolletor and rename package to main.

Reply:
_Sorry for hijacking this comment, I should have clicked a line above._